tests/routes/setup_search_tests.py

```diff
@@ -22,16 +22,16 @@
     Slug,
     Variant,
 )
-from app.db.models.law_policy.metadata import (
+from db_client.models.law_policy.metadata import (
     FamilyMetadata,
 )
-from app.db.models.law_policy import Geography
-from app.db.models.document.physical_document import (
+from db_client.models.law_policy import Geography
+from db_client.models.document.physical_document import (
     LanguageSource,
     PhysicalDocument,
     PhysicalDocumentLanguage,
 )
-from app.initial_data import run_data_migrations
+from db_client.initial_data import run_data_migrations
```

Comment:
See Ruff/Black comment

Reply:
Let's have a look together later pelase. We will merge and test this in Staging for now.